tests/test_seaexplorer.py

```diff
@@ -58,12 +58,12 @@ def test_merge_rawnc():
     result_default = seaexplorer.merge_parquet(
             'tests/data/realtime_rawnc/',
             'tests/data/realtime_rawnc/',
-            example_dir / 'example-seaexplorer/deploymentRealtime.yml')
+            str(example_dir / 'example-seaexplorer/deploymentRealtime.yml'))
```

Comment:
This file still has the `str` conversions as well....  We shouldn't need these.  Thanks!